flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/Constants.java

```diff
@@ -36,12 +36,12 @@ public class Constants {
     public static final String BALLERINAX = "ballerinax";
     // AI related constants
     public static final String AI = "ai";
-    public static final String OPENAI_MODEL_VERSION = "1.1.0";
-    public static final String OLLAMA_MODEL_VERSION = "1.0.1";
-    public static final String AZURE_MODEL_VERSION = "1.0.1";
-    public static final String MISTRAL_MODEL_VERSION = "1.0.1";
-    public static final String ANTHROPIC_MODEL_VERSION = "1.0.1";
-    public static final String DEEPSEEK_MODEL_VERSION = "1.0.1";
+    public static final String OPENAI_MODEL_VERSION = "1.2.0";
```

Comment:
If we do a patch release do we need to come and update the version here?

Reply:
Yes, that's the current behaviour. We need to find a better way, though. @NipunaRanasinghe how do we handle this for other modules? Do we automatically use the new version of them? R we looking into `dependencies.toml` as well?

Reply:
I think we can avoid passing version information altogether in createModelObject. This may all to pull latest patch version. can you check ?

Reply:
We need to discuss it first, right?. What behaviour should we follow. Automatic new version pull also affects the version assertions in the test files.

Reply:
We can update the tests to ignore the version flag. However, I feel that doing a separate LS release for every patch version isn't really necessary. I believe we follow a similar approach when pulling connectors as well. Hope that’s correct, @nipunayf.

Reply:
The function data builder handles the complexity of selecting the appropriate version after a component is chosen during the search. If a version is not specified, it fetches the latest compatible version based on the current Ballerina version, as provided by Ballerina Central.

Therefore, for the initial call, it’s best to omit the version to retrieve the latest one. Once the module is added to the package, the version can be specified to avoid unnecessary calls to Central.

This logic only applies to forms (after selecting a component). Unfortunately, Central is not integrated into the search, and we rely on the local index. As a result, to generate the icon URL during the search, the version must be explicitly provided.